ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-devel__downstream-candidate.yaml

```diff
@@ -53,31 +53,9 @@ tests:
       TEST_FILTERS: ~DisconnectedOnly&;~Disruptive&
       TEST_RELEASE_TYPE: Pre-GA
       TEST_SCENARIOS: sig-kata.*Kata Author
-      TEST_TIMEOUT: "75"
+      TEST_TIMEOUT: "90"
       WORKLOAD_TO_TEST: peer-pods
     workflow: sandboxed-containers-operator-e2e-azure
-- as: azure-ipi-coco
-  cron: 0 0 31 2 1
-  steps:
-    cluster_profile: azure-qe
-    env:
-      BASE_DOMAIN: qe.azure.devcluster.openshift.com
-      CATALOG_SOURCE_IMAGE: quay.io/redhat-user-workloads/ose-osc-tenant/osc-test-fbc:1.10.0-1753281667
-      CATALOG_SOURCE_NAME: brew-catalog
-      ENABLEPEERPODS: "true"
-      EXPECTED_OPERATOR_VERSION: 1.10.0
-      INSTALL_KATA_RPM: "true"
-      KATA_RPM_VERSION: 3.17.0-3.rhaos4.19.el9
-      RUNTIMECLASS: kata-remote
-      SLEEP_DURATION: "0"
-      TEST_FILTERS: ~DisconnectedOnly&;~Disruptive&
-      TEST_RELEASE_TYPE: Pre-GA
-      TEST_SCENARIOS: sig-kata.*Kata Author
-      TEST_TIMEOUT: "75"
-      TRUSTEE_CATALOG_SOURCE_IMAGE: quay.io/redhat-user-workloads/ose-osc-tenant/trustee-fbc-4-19:trustee-fbc-4-19-on-push-76wsv-build-image-index
-      TRUSTEE_CATALOG_SOURCE_NAME: trustee-catalog
-      WORKLOAD_TO_TEST: coco
-    workflow: sandboxed-containers-operator-e2e-azure
```

Comment:
Why is this commit removing job that was introduced on the previous commit? It'd be perhaps easier to follow if you rebased and not included it in the first place... (anyway thanks for extracting the timeout change into another commit, that's definitely step in the right direction)